# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/moz.configure
# Commit: 5147903580b8
# Full Hash: 5147903580b8f39197d67bddd5a0fb91d3670da4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-27 04:32:10
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r 0aad3393b572 -r 5147903580b8 toolkit/moz.configure
--- a/toolkit/moz.configure	Mon Feb 26 16:07:07 2024 +0000
+++ b/toolkit/moz.configure	Mon Feb 26 16:07:08 2024 +0000
@@ -2377,17 +2377,6 @@
 
 # FFmpeg's ffvpx configuration
 # ==============================================================
-with only_when(compile_environment):
-
-    @depends(target)
-    def libav_fft(target):
-        if target.os == "Android" and target.cpu != "arm":
-            return True
-        return target.kernel in ("WINNT", "Darwin") or target.cpu == "x86_64"
-
-    set_config("MOZ_LIBAV_FFT", depends(when=libav_fft)(lambda: True))
-    set_define("MOZ_LIBAV_FFT", depends(when=libav_fft)(lambda: True))
-
 
 # Artifact builds need MOZ_FFVPX defined as if compilation happened.
 with only_when(compile_environment | artifact_builds):
